ext/hash/hash.stub.php

```diff
@@ -17,33 +17,33 @@ function hash_update(HashContext $context, string $data): bool {}
 /** @param resource $handle */
 function hash_update_stream(HashContext $context, $handle, int $length = -1): int {}
```

Comment:
```suggestion
function hash_update_stream(HashContext $context, $stream, int $length = -1): int {}
```
Would be my suggestion for the general name of stream resources.